spinalcordtoolbox/utils/shell.py

```diff
@@ -199,8 +199,10 @@ def error(self, message):
         """
         # Source: https://stackoverflow.com/a/4042861
         self.print_help(sys.stderr)
-        printv(f'\n{self.prog}: error: {message}\n', verbose=1, type='error', file=sys.stderr)
-        sys.exit(2)
+        message_formatted = (ANSIColors16.Bold + ANSIColors16.LightRed
+                             + f'\n{self.prog}: error: {message}\n\n'
+                             + ANSIColors16.ResetAll)
+        self.exit(2, message_formatted)
```

Comment:
NB: The formatting change here was done in an effort to better mimic the parent method from `argparse`:

```python
    def error(self, message):
        """error(message: string)

        Prints a usage message incorporating the message to stderr and
        exits.

        If you override this in a subclass, it should not return -- it
        should either exit or raise an exception.
        """
        self.print_usage(_sys.stderr)
        args = {'prog': self.prog, 'message': message}
        self.exit(2, _('%(prog)s: error: %(message)s\n') % args)
```

Reply:
Additionally, I found that I needed to add a second `\n` in order to preserve the same formatting as before. 

I'm not sure why that is? Maybe it's because `printv` inherently adds a newline to the end of the message, since it's a print function, but `self.exit` does not? Not sure, tut I thought I'd acknowledge the change just in case it's not, say, consistent across platforms.

Reply:
About the underscore: yes, it's even documented that `gettext.gettext` is often imported as simply `_` ([link](https://docs.python.org/3/library/gettext.html#gettext.gettext)). It's not my favourite convention though, since it's so sneaky! But I believe this is a cross-language convention (for example C and PHP both seem to use it), so it's not going away.